addons/mail/static/src/new/utils.js

```diff
@@ -56,19 +48,36 @@ export function onExternalClick(refName, cb) {
 
 export function useAutoScroll(refName) {
     const ref = useRef(refName);
-    let isScrolled = false;
-    onMounted(() => {
-        ref.el.scrollTop = ref.el.scrollHeight;
-    });
-    onWillPatch(() => {
-        const el = ref.el;
+    let el = null;
```

Comment:
I think I would prefer using `ref.el` everywhere rather than syncing an extra variable.

Reply:
Actually even when not using a local `ref` and just `ref.el`, there's still a risk that the `ref.el` changes during patching while current component is still mounted.

====================

It might just be a dumb idea, but I think It would have been nice to have a generic hook like `useOnRef(refName, attributeName, value);` that is totally equivalent to setting an attribute in the template but in a "hooky" way.

For example, for the `add/removeListener` on `"scroll"` below, it could have just be:
```js
usOnRef(refName, 't-on-scroll', onScroll);
```
This would be much more robust, as under-the-hood it's a `t-on-scroll` and totally handle local mount/unmount of ref.